tests/test_requirements.txt

```
@@ -1,15 +1,15 @@
-astropy==3.0.5
+astropy==3.2.3
 codecov==2.0.15
-coverage==4.5.1
+coverage==4.5.4
 fastcache==1.0.2
 glueviz==0.13.3
 h5py==2.10.0
 ipython==7.1.1
-matplotlib==3.1.3
+matplotlib==3.3.0
```

For now it's alright but maybe we should future proof this and use their bug fixes releases as soon as they come out ?
```suggestion
matplotlib~=3.3
```